### DISCUSSION TITLE
Popular posts/top posts

### DISCUSSION THREAD

**Discussion by jxxe** [statamic/cms#6834] [Q&A]
Hi!

I'm wondering what the community thinks would be the best way to implement a top/popular/most viewed posts feature. The view counts don't have to be exact, just produce plausible results.

To manage traffic, I could employ tactics like adding a 10% chance to store 10 views for an entry when it's loaded, but where? How could that work with static pages? Is there a relatively easy API for Google Analytics I could use? Or a dedicated service? I suppose it wouldn't be too bad to build my own?

**Comment by edalzell** [comment]
I'd recommend utilizing your analytics package's API for this.
